data_validation/result_handlers/text.py

```diff
@@ -35,6 +36,17 @@ def print_formatted_(self, result_df):
         Utility for printing formatted results
         :param result_df
         """
+        # the text transformer chokes on bytestring results (e.g. SHA256) this
+        # dataframe slice is to remove the source and target values
+        if "validation_type" in result_df.columns:
```

Comment:
Won't every validation fall under this if loop? Assuming you only want to change row validation result handlers.

Reply:
this was actually to bypass the unit tests for the text handler which uses pandas series...I'm open to a better solution.

Reply:
Ah I see.. I would instead add a 'validation_type' column within the unit test sample pandas series